container-images/tcib/base/os/tobiko/run_tobiko.sh

```diff
@@ -13,6 +15,7 @@ fi
 
 # set default values for the required variables
 TOBIKO_VERSION=${TOBIKO_VERSION:-master}
+TOBIKO_PRIVATE_KEY_FILE=${TOBIKO_PRIVATE_KEY_FILE:-id_ecdsa}
 
 # export OS_CLOUD variable
 [ ! -z ${TOBIKO_OS_CLOUD} ] && export OS_CLOUD=${TOBIKO_OS_CLOUD} || export OS_CLOUD=default
```

Comment:
This is just a small thing I noticed. It is not related to the PR but I wanted to at least mention it.
```bash
[ ! -z ${TOBIKO_NUM_PROCESSES} ] && export=TOX_NUM_PROCESSES=${TOBIKO_NUM_PROCESSES}
```
The '=' should not probably be here.

Reply:
good catch, thanks!